static/css/bureau-structure.less

```diff
@@ -1,51 +1,56 @@
+/* ==========================================================================
+   cfgov-refresh
+   Bureau structure page at /the-bureau/bureau-structure/.
+   ========================================================================== */
+
 .org-chart {
-    
-    .respond-to-min(600, {
+
+    .respond-to-min(600px, {
         width: 530px;
         margin: 0 auto;
     });
-    
-    .respond-to-min(1024, {
+
+    .respond-to-min(1024px, {
         width: 660px;
     });
-    
+
     &_root {
         margin-left: auto;
         margin-right: auto;
         text-align: center;
     }
-    
+
     &_branches,
     &_nodes {
         list-style-type: none;
         margin: 0;
         padding: 0;
     }
-    
-    /* On larger screen sizes, the lines of the org chart are drawn with before & after 
+
+    /* On larger screen sizes, the lines of the org chart are drawn with before & after
        pseudo elements applied to .org-chart_branches & .org-chart_branch.
      */
     &_branches {
         padding-top: unit(@grid_gutter-width, px);
-        
+
         /* :before pseudo element's left border draws a centered vertical line
            down from .org-chart_root to .org-chart_branches.*/
         .respond-to-min(600px, {
             position: relative;
-            
+
             &:before{
                 content: '';
-                position: absolute; 
-                top: 0; 
+                position: absolute;
+                top: 0;
                 left: 50%;
                 border-left: 1px solid @gray-50;
-                width: 0; 
+                width: 0;
                 height: unit(@grid_gutter-width, px);
```

Comment:
[`@grid_gutter-width`](https://github.com/cfpb/cf-grid/blob/gh-pages/src/cf-grid.less#L42) is already in pixels.